### ISSUE TITLE
heroku build and deploy upon push to master

### ISSUE THREAD

**Issue by garyspencer** [boulderReactCodingNight/03-react-twitter#43]

**Comment by garyspencer** [comment]
@jenniferwiner is it ok to set up on a personal heroku account for now?

**Comment by garyspencer** [comment]
https://brcn-03-react-twitter.herokuapp.com/

**Comment by jenniferwiner** [comment]
@garyspencer Yeah, all good to set up on a personal account. Makes sense. I get that error when I start up the client before the server.

**Comment by garyspencer** [comment]
@jenniferwiner I hit a road block getting the client+server to work in heroku, still getting 404s even after populating the db on heroku.  I did eventually add scripts that seed & migrate targetting the heroku postgres database, but still had 404 issues..  would love to work together if possible
